test/transforms.jl

```diff
@@ -859,6 +859,32 @@
     @test rt == rtₒ
   end
 
+  @testset "RowTable" begin
+    a = [3, 2, 1, 4, 5, 3]
+    b = [1, 4, 4, 5, 8, 5]
+    c = [1, 1, 6, 2, 4, 1]
+    t = Table(; a, b, c)
+    T = RowTable()
+    n, c = apply(T, t)
+    tₒ = revert(T, n, c)
+    @test typeof(n) <: Vector
+    @test Tables.rowaccess(typeof(n))
```

Comment:
Do we really need the typeof here @ceferisbarov ?

Reply:
From Tables.jl API:  
_

Tables.rowaccess(table)    Declare that your table type defines a Tables.rows(table) method_  
  
This seems to be the intended use, but it also works when I remove `typeof`.